editor/src/components/canvas/ui-jsx.test-utils.tsx

```diff
@@ -165,6 +165,7 @@ export interface EditorRenderResult {
   getRecordedActions: () => ReadonlyArray<EditorAction>
 }
 
+// FIXME This should be calling formatTestProjectCode
```

Comment:
Is there a reason to not do that?

Reply:
This is only because I want to make that change in a separate PR so that it doesn't pollute this diff